### ISSUE TITLE
"multi-word-component-names" doesn't understand compound words

### ISSUE THREAD

**Issue by charles-allen** [vuejs/eslint-plugin-vue#1699]
**What rule do you want to change?**

- [multi-word-component-names](https://github.com/vuejs/eslint-plugin-vue/pull/1661)

**Does this change cause the rule to produce more or fewer warnings?**

- Fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**

- Better dictionary?

**Please provide some example code that this change will affect:**

`overview.vue`
```vue
<template>
  <div>...</div>
</template>
<script src="./overview.ts" lang="ts"></script>
```

**What does the rule currently do for this code?**

```
.../packages/website/src/overview/overview.vue
  1:1  error  Component name "overview" should always be multi-word  vue/multi-word-component-names
```

**What will the rule do after it's changed?**

- No error

**Additional context**

- ["overview" is a word...](https://www.google.com/search?q=define+overview)

**Comment by charles-allen** [comment]
I misunderstood the error message...
